doc/manual/src/contributing/meson.build

```diff
@@ -0,0 +1,15 @@
+# Intermediate step for experimental-feature-descriptions.md.
+# This splorks the output of generate-xp-features.nix as JSON,
+# which gets written as a directory tree below.
```

Comment:
Looks complete to me.
Also probably best to avoid slang, because it raises the difficulty and it changes and expires.

```suggestion
# This runs generate-xp-features.nix to generate experimental-feature-descriptions.md
```